lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -2123,8 +2132,11 @@ int mme_app_handle_initial_paging_request(
     mme_ue_context_dump_coll_keys(&mme_app_desc_p->mme_ue_contexts);
     OAILOG_FUNC_RETURN(LOG_MME_APP, RETURNerror);
   }
-  return mme_app_paging_request_helper(
-      ue_context_p, true, true /* s-tmsi */, CN_DOMAIN_PS);
+  if (!(ue_context_p->paging_retx_count)) {
```

Comment:
Since this is handling the initial paging request, we should log an error if paging_retx_count is not zero.

Reply:
Before mme restarts, UE is in idle mode, so on arrival of DL data, Paging Indication is sent to mme and mme sends Paging message to eNB and starts timer.
After mme restarts again mme receives Paging Indication which should not lead to sending Paging message to eNB again. So I added this check. 
 Should we still log an error if paging_retx_count is not zero?